ui/blocks/LastCheckin.tsx

```diff
@@ -16,6 +17,7 @@ interface Props {
 
 export const LastCheckin: React.FC<Props> = ({ checkin, area, onCheckout }) => {
   const checkedOut = !!checkin.leftAt
+  const guestInfo = useCurrentGuest()
```

Comment:
What if a user didn't check the remember me checkbox? Then `useCurrentGuest()` doesn't show anything, if I remember correctly